tools/flaky-test-retryer/github_commenter.go

```diff
@@ -118,30 +123,40 @@ func (gc *GithubClient) getOldComment(org, repo string, pull int) (*github.Issue
 
 // parseEntries collects retry information from the given comment, so we can reuse it in
 // a new comment.
-func parseEntries(comment *github.IssueComment) (map[string]int, error) {
-	entries := map[string]int{}
+func parseEntries(comment *github.IssueComment) (map[string]*entry, error) {
+	entries := make(map[string]*entry)
```

Comment:
does make need a second arg for len?

Reply:
seems okay in my testing, we make a new struct each time we add to it